azure/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@microsoft/vscode-azext-azureutils",
     "author": "Microsoft Corporation",
-    "version": "0.3.4",
+    "version": "0.3.5",
```

Comment:
This version number needs to be updated in `package-lock.json` as well. It appears in two places near the top of the document.

Reply:
I will update it over there as well. Thanks!

Reply:
You should just run `npm install` as it will handle that for you.